.travis.yml

```diff
@@ -33,21 +33,6 @@ matrix:
     env: PUPPET_VERSION="~> 4.0" CHECK=test
   - rvm: 2.4.0-preview1
     env: PUPPET_VERSION="~> 4.0" CHECK=test
-  - rvm: 2.3.1
```

Comment:
That's not a local modulesync config difference, is it? If so, that should be handled at the modulesync_config level. 

However, I think we should keep 2.3.1.